### PR TITLE
Add git.

### PR DESCRIPTION
**Pull request by dmatthew**

### CODE REVIEW

**Comment by dmatthew** [comment]
git-all is a meta package which installs about 200 additional packages. Is that what we want?

**Comment by colby** [comment]
```
~/src/Work/COP/cookbooks/base / bugfix/add-git
$ kli
Instance             Driver   Provisioner  Verifier  Transport  Last Action
default-ubuntu-1204  Vagrant  ChefZero     Busser    Ssh        Verified
default-ubuntu-1404  Vagrant  ChefZero     Busser    Ssh        Verified
default-ubuntu-1604  Vagrant  ChefZero     Busser    Ssh        Verified
default-centos-68    Vagrant  ChefZero     Busser    Ssh        Verified
default-centos-72    Vagrant  ChefZero     Busser    Ssh        Verified
default-debian-711   Vagrant  ChefZero     Busser    Ssh        Verified
default-debian-86    Vagrant  ChefZero     Busser    Ssh        Verified
```

File: recipes/variables.rb [hunk 1]
```diff
@@ -1,9 +1,16 @@
 file '/etc/environment' do
-    action :touch
+    owner  'root'
+    group  'root'
+    mode   0666
```
Comment (hardyjohnson) [inline recipes/variables.rb]: 👿

File: test/integration/default/serverspec/default_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  end
+
+  describe package('git') do
+    it { should be_installed }
+  end
+
+  describe file('/etc/environment') do
+    it { should be_owned_by 'root' }
+    it { should be_grouped_into 'root' }
+    it { should be_mode '644' }
```
Comment (hardyjohnson) [inline test/integration/default/serverspec/default_spec.rb]: This should fail.

Comment (colby) [inline test/integration/default/serverspec/default_spec.rb]: Ah, it should be 644. I've made the change in e94db9f

**Pull request merged**